### PR TITLE
RgueCmdr audio updates

### PR DESCRIPTION
**Pull request by NoSloppy**
bgnlock: Created from existing lock and additional sounds.

lock: Edited to loop cleaner. Edited lock to have less monophonic-like hum mixed in and added some electrical arcing.

hum: High passed @100Hz and normalized hum to -5db. This is in lieu of editing config.ini volHum parameter, as (initially) config.ini didn't exist. 
Better mix balance now and falls in line with the majority of other font levels.

out: normalized to 0dB, full fade out at end.

clsh: Removed excess silences and normalized to 0dB. clsh09 also HP @100Hz.

### CODE REVIEW

**Comment by NoSloppy** [comment]
Replaced highpassed files.

File: RgueCmdr/license.txt [hunk 1]
```diff
@@ -1 +1,11 @@
-These files are free to use, modify, and redistribute without attribution or any other restrictions.
\ No newline at end of file
+These files are free to use, modify, and redistribute without attribution or any other restrictions.
+
+Modifications by Brian Conner 2022:
+Detailed changes can be found in this font's ReadMe.txt.
+Any additional sounds sourced from creative-commons licensed freesound.org.
```
Comment (profezzorn) [inline RgueCmdr/license.txt]: specify files

Comment (NoSloppy) [reply RgueCmdr/license.txt]: Done.

**Comment by profezzorn** [comment]
> hum: High passed @100hz 

Is this still the case?

**Comment by NoSloppy** [comment]
> > hum: High passed @100hz
> 
> Is this still the case?

Nope.

**Pull request merged**